Fix build warnings

From CI

```
MSBUILD : warning CA1405: Microsoft.Interoperability : 'PasswordBoxHintAdorner' is marked ComVisible(true) but has the following ComVisible(false) types in its object hierarchy: 'Adorner', 'FrameworkElement', 'UIElement', 'Visual', 'DependencyObject', 'DispatcherObject' [C:\projects\git-credential-manager-for-windows\Cli-AskPass\Cli-Askpass.csproj]
  Code Analysis Complete -- 0 error(s), 1 warning(s)
          1 file(s) copied.
          1 file(s) copied.
  The system cannot find the file specified.
          1 file(s) copied.
  The system cannot find the file specified.
          1 file(s) copied.
          1 file(s) copied.
  The system cannot find the file specified.
          1 file(s) copied.
  The system cannot find the file specified.
          1 file(s) copied.
          1 file(s) copied.
```

--- a/Cli-Askpass/PasswordBoxHintAdorner.cs
+++ b/Cli-Askpass/PasswordBoxHintAdorner.cs
@@ -6,6 +6,7 @@ using System.Windows.Media;
 
 namespace Microsoft.Alm.Gui
 {
+    [System.Runtime.InteropServices.ComVisible(false)]
     public class PasswordBoxHintAdorner : Adorner
     {
         public PasswordBoxHintAdorner(UIElement adornedElement, string hintText, Style hintStyle, VisibilityDelegate visibilityCallback)

